packages/ibm/system-connection-details.tsx

```diff
@@ -34,7 +36,26 @@ import {
   getFlashSystemSecretName,
 } from './utils';
 
-const VOLUME_MODES = ['thick', 'thin'];
+const VOLUME_MODES_VALUES = [
+  'thick',
+  'thin',
+  'compressed',
+  'dedulipcated',
```

Comment:
is spelling wrong on purpose ??
```suggestion
  'deduplicated',
```